airflow/example_dags/example_http_operator.py

```diff
@@ -83,7 +83,7 @@
     http_conn_id='http_default',
     endpoint='',
     params={},
-    response_check=lambda response: True if "Google" in response.content else False,
+    response_check=lambda response: True if b"Google" in response.content else False,
```

Comment:
My bad, I didn't fully understand the details of the problem. Digging into it, we can use the requests library's response object to handle the decoding:

```python
lambda response: 'Google' in response.text
```

should do what we want